instagram-app/src/components/SearchBar.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './SearchBar.css';
+
+  class SearchBar extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        input: ""
+      };
+    }
```

Comment:
It would be better to place this input variable in your application state so that you can use it in other components.